src/Models/BaseElement.php

```diff
@@ -128,6 +130,14 @@ class BaseElement extends DataObject
      */
     private static $disable_pretty_anchor_name = false;
 
+    /**
+     * Set to false to prevent an in-line edit form from showing in an elemental area. Instead the element will be
+     * clickable and a GridFieldDetailForm will be used.
+     *
+     * @var bool
```

Comment:
Add `@config`- at some stage we'll be using this annotation for the API docs site